english/indian_winter/winters_in_india.txt

```diff
@@ -0,0 +1 @@
+We've all heard about the Indian Summers, here's a take on its winters.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.